chapter02/sdt/ex1.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+use Cwd qw( abs_path );
+use File::Spec qw( curdir );
+
+opendir(DIR, File::Spec->curdir()) or die $!;
```

Comment:
``curdir`` seems useful. Probably for this case ``"."`` was enough, but modules!